docker/oms-events/Dockerfile.dev

```diff
@@ -9,17 +9,17 @@ RUN mkdir -p /usr/src/shared
 #WORKDIR /usr/src/app
 
 RUN mkdir -p /usr/app/ \
-        && mkdir -p /usr/app/frontend \
         && mkdir -p /usr/app/media \
         && mkdir -p /usr/scripts \
-        && mkdir -p /usr/shared
+        && mkdir -p /usr/shared \
+        && mkdir -p /usr/app/oms-events
 
-WORKDIR /usr/app
+WORKDIR /usr/app/oms-events
 
 ADD ./configFile.json /usr/scripts/configFile.json
 
-CMD cp /usr/scripts/configFile.json /usr/app/lib/config/configFile.json && \
+CMD cp /usr/scripts/configFile.json lib/config/configFile.json && \
```

Comment:
Can we use `/usr/app/scripts` for it? Also I suggest not copying it, but linking instead.

Reply:
Looking at this now (I mostly just copied it before) it makes me wonder why this is copied at the command on launching, why not earlier using `ADD` or something.